rpc-protocol/src/main/java/org/sonarsource/sonarlint/core/rpc/protocol/backend/initialize/InitializeParams.java

```diff
@@ -88,6 +89,7 @@ public InitializeParams(
    * @param sonarlintUserHome           Path to SonarLint user home directory. If null, will default to ~/.sonarlint
    * @param standaloneRuleConfigByKey   Local rule configuration for standalone analysis. This configuration will override defaults rule activation and parameters.
    */
+  @Deprecated(since = "10.4")
```

Comment:
Should we create a ticket to go through the deprecated methods during hardening? We now have the third very large constructor in this class.

Reply:
Good point. Created: https://sonarsource.atlassian.net/browse/SLCORE-914